package.json

```diff
@@ -18,8 +18,9 @@
   },
   "homepage": "https://github.com/contexte/cahouete#readme",
   "devDependencies": {
-    "babel-core": "6.26.3",
-    "babel-preset-env": "1.7.0",
+    "@babel/core": "7.5.5",
```

Comment:
Shoudn't we take the opportunity of this small project to experiment Dependabot on our JS? Meaning setting wildcard versions in `package.json` and let a bot do the upgrade job?

Reply:
Let's say it's too expensive right now and make it a goal in the next weeks/months.